ImmichFrame update script issue

### ✅ Have you read and understood the above guidelines?

yes

### 🔎 Did you run the script with verbose mode enabled?

Yes, verbose mode was enabled and the output is included below

### 📜 What is the name of the script you are using?

ImmichFrame with update command in LXContainer

### 📂 What was the exact command used to execute the script?

update

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 13

### 📈 Which Proxmox version are you on?

pve-manager/9.1.9/ee7bad0a3d1546c9 (running kernel: 7.0.0-3-pve)

### 📝 Provide a clear and concise description of the issue.

Hi,
I launch the command 'update' in the container of proxmox. The LXContainer comes from the PVE help script.

### 🔄 Steps to reproduce the issue.

You installed 1.32.0 version of Immichframe via PVE help script (it creates the LXC with ImmichFrame)
You launch the command update

### ❌ Paste the full error output (if available).

Update available: immichframe 1.0.32.0 → 1.0.33.0
  ✔️   Stopped Service
  ✔️   Backed up Configuration
  ✔️   Deployed: immichframe (1.0.33.0)
  ⏳   Setting up ImmichFrameThe command could not be loaded, possibly because:
  * You intended to execute a .NET application:
      The application 'publish' does not exist.
  * You intended to execute a .NET SDK command:
      No .NET SDKs were found.

Download a .NET SDK:
https://aka.ms/dotnet/download

Learn about SDK resolution:
https://aka.ms/dotnet/sdk-not-found
  ✖️   in line 45: exit code 145 (Unknown error): while executing command  dotnet publish ImmichFrame.WebApi/ImmichFrame.WebApi.csproj --configuration Release --runtime linux-x64 --self-contained false --output /opt/immichframe

  --- Last 20 lines of log ---
[2026-04-12 11:00:39] [INFO] Checking for update: immichframe
[2026-04-12 11:00:40] [OK] Update available: immichframe 1.0.32.0 → 1.0.33.0
[2026-04-12 11:00:40] [INFO] Stopping Service
[2026-04-12 11:00:40] [OK] Stopped Service
[2026-04-12 11:00:40] [INFO] Backing up Configuration
[2026-04-12 11:00:40] [OK] Backed up Configuration
[2026-04-12 11:00:40] [INFO] Fetching GitHub release: immichframe (1.0.33.0)
[2026-04-12 11:00:41] [OK] Deployed: immichframe (1.0.33.0)
[2026-04-12 11:00:41] [INFO] Setting up ImmichFrame
[2026-04-12 11:00:41] [ERROR] in line 45: exit code 145 (Unknown error): while executing command  dotnet publish ImmichFrame.WebApi/ImmichFrame.WebApi.csproj --configuration Release --runtime linux-x64 --self-contained false --output /opt/immichframe
  -----------------------------------

### 🖼️ Additional context (optional).

_No response_

--- a/install/immichframe-install.sh
+++ b/install/immichframe-install.sh
@@ -43,8 +43,6 @@ cd /tmp/immichframe/immichFrame.Web
 $STD npm ci
 $STD npm run build
 cp -r build/* /opt/immichframe/wwwroot
-$STD apt remove -y dotnet-sdk-8.0
-$STD apt autoremove -y
 rm -rf /tmp/immichframe
 mkdir -p /opt/immichframe/Config
 curl -fsSL "https://raw.githubusercontent.com/immichFrame/ImmichFrame/main/docker/Settings.example.yml" -o /opt/immichframe/Config/Settings.yml

